go-controller/pkg/clustermanager/pod/allocator_test.go

```diff
@@ -545,6 +594,22 @@ func TestPodAllocator_reconcileForNAD(t *testing.T) {
 				ipamClaimsReconciler,
 			)
 
+			nadLister := v1nadmocks.NetworkAttachmentDefinitionLister{}
+			nadNamespaceLister := v1nadmocks.NetworkAttachmentDefinitionNamespaceLister{}
+			nadLister.On("NetworkAttachmentDefinitions", "namespace").Return(&nadNamespaceLister)
+			mockedNADs := []*nadapi.NetworkAttachmentDefinition{}
+			for _, nad := range tt.args.nads {
+				if nad.Namespace == "namespace" {
+					mockedNADs = append(mockedNADs, nad)
+				}
+			}
+			nadNamespaceLister.On("List", labels.Everything()).Return(mockedNADs, nil)
+
+			fakeRecorder := record.NewFakeRecorder(10)
```

Comment:
nit: maybe move this to an helper.

Reply:
We are doing this only once on this test, I don't want to create a helper and modify other packages.